README.md

```diff
@@ -45,15 +45,17 @@ corresponds to the 1.6 release. The "master" branch will normally contain
 development for the next connector release in progress.
 
 Currently, the following branches are actively supported: 
-3.4.x ([master](https://github.com/datastax/spark-cassandra-connector/tree/master)),
+3.5.x ([master](https://github.com/datastax/spark-cassandra-connector/tree/master)),
+3.4.x ([b3.4](https://github.com/datastax/spark-cassandra-connector/tree/master/b3.4)),
```

Comment:
Please remove `master` here, it should be  `https://github.com/datastax/spark-cassandra-connector/tree/b3.4`